ocs_ci/utility/localstorage.py

```diff
@@ -32,3 +38,122 @@ def fetch_all_device_paths():
     os.chdir(constants.TOP_DIR)
     shutil.rmtree(path)
     return dev_paths
+
+
+def get_new_device_paths(device_sets_required, osd_size_capacity_requested):
+    """
+    Get new device paths to add capacity over Baremetal cluster
+
+    Args:
+        device_sets_required (int) : Count of device sets to be added
+        osd_size_capacity_requested (int) : Requested OSD size capacity
+
+    Returns:
+        list : List containing added device paths
+
+    """
+    ocp_obj = OCP()
+    workers = get_typed_nodes(node_type="worker")
+    worker_names = [worker.name for worker in workers]
+    output = ocp_obj.exec_oc_cmd("get localvolume local-block -n local-storage -o yaml")
+    cur_device_list = output["spec"]["storageClassDevices"][0]["devicePaths"]
+    path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+    utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+    os.chdir(path)
+    utils.run_cmd("ansible-playbook devices_by_id.yml")
+    with open("local-storage-block.yaml", "r") as cloned_file:
+        with open("local-block.yaml", "w") as our_file:
+            device_from_worker1 = device_sets_required
+            device_from_worker2 = device_sets_required
+            device_from_worker3 = device_sets_required
```

Comment:
Please use a dynamic arg for the vars representing the workers. You can use config.ENV_DATA['worker_replicas']
for that

Reply:
Will fix that!